src/user/interface.ts

```diff
@@ -0,0 +1,21 @@
+import {Journal} from '../journal/type'
+import  {Entry, EntryMutable} from "../entry/type";
+import {Tag, TagDescription} from '../tag/type'
+
+export interface ActionResponse<T> {
```

Comment:
I think this one should be called `Response` and should also be implemented for main calls inside `src/client/index.ts`